src/layouts/wrapper.tsx

```diff
@@ -3,13 +3,24 @@ import Footer from '@layout/footer';
 import Header from '@layout/header';
 import ScrollToTop from '@ui/scroll-to-top';
 import { useTheme } from 'next-themes';
+import { useEffect } from 'react';
+import { selectAuthState } from '../features/auth/store/auth.slice';
+import useAppDispatch from '../store/hooks/useAppDispatch';
+import { findLikedItemsByAccount } from '../features/account/store/account.actions';
+import useAppSelector from '../store/hooks/useAppSelector';
 
 type Props = {
   children: React.ReactNode;
 };
 
 const Wrapper = ({ children }: Props) => {
   const { theme } = useTheme();
+  const dispatch = useAppDispatch();
+  const { isAuthenticated, address } = useAppSelector(selectAuthState);
+
+  useEffect(() => {
+    dispatch(findLikedItemsByAccount(address));
```

Comment:
Pay attention here. You are not validating wether or not the user is authenticated. Please add a condition and make the request only for authenticated users